src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -64,55 +62,26 @@
  * about these suffixes are added.
  */
 public class SkroutzGreekStemmer {
-  public final static String DEFAULT_STOPWORD_FILE = "stopwords.txt";
-  private final CharArraySet stopwords;
-  private Map<String, char[]> stepZeroExceptions;
+  private CharArraySet protectedWords = CharArraySet.EMPTY_SET;
+  private CharArrayMap<char[]> stepZeroExceptions = CharArrayMap
+          .emptyMap();
 
-  public SkroutzGreekStemmer(final CharArraySet stopwords) {
-    this.stopwords = stopwords;
+  public SkroutzGreekStemmer() {
     try {
+      this.protectedWords = loadProtected("protected_words");
       this.stepZeroExceptions = loadExceptions("step_0_exceptions");
     } catch(Exception ex) {
       System.err.println("Failed to load exceptions from yaml file");
```

Comment:
This is not logged anywhere, it's just for debugging purposes. We currently don't do any logging. Should we add some?